File: src/main/java/com/travel/community/pages/account/dto/response/LoginResponse.java
package com.travel.community.pages.account.dto.response;

import lombok.Builder;
import lombok.Getter;

/**
 * 로그인 성공 사용자 정보 반환 객체
 * LoginResponse
 */
@Getter
@Builder
public class LoginResponse {
    private String userId; // 사용자 ID
    private String nickName; // 닉네임
    private String email; // 이메일
}

File: vue/src/pages/login/types/response/LoginResponse.ts
/**
 * 로그인 성공시 사용자 정보 반환 객체
 */
export interface LoginResponse {
	userId: string; // 사용자 ID
	nickName: string; // 사용자 닉네임
	email: string; // 사용자 이메일
}
